### ISSUE TITLE
Add color picker field

### ISSUE THREAD

**Issue by wpnzach** [wpninjas/ninja-forms#3198]
Add color picker field

**Comment by agenceblink** [comment]
Yes, the color picker really misses ninjaform to allow webmasters to create order forms that allow to customize the color of the product for example. The color Picker exists on Gravity Form, really surprising that it is missing on ninjaForm!
